python/ray/worker.py

```diff
@@ -372,14 +373,18 @@ def sigterm_handler(signum, frame):
         sys.exit(0)
 
 
-def get_gpu_ids():
+def get_gpu_ids(as_str=False):
     """Get the IDs of the GPUs that are available to the worker.
 
     If the CUDA_VISIBLE_DEVICES environment variable was set when the worker
     started up, then the IDs returned by this method will be a subset of the
     IDs in CUDA_VISIBLE_DEVICES. If not, the IDs will fall in the range
     [0, NUM_GPUS - 1], where NUM_GPUS is the number of GPUs that the node has.
 
+    Args:
+        as_str (Boolean): If true, return gpu ids in string format. By default,
+            it is False.
```

Comment:
Add a comment saying that this will be changed to default to True in the future.

Reply:
Done